cuda_core/cuda/core/experimental/_memory.pyx

```diff
@@ -816,13 +856,19 @@ class DeviceMemoryResource(MemoryResource):
         -------
             The shareable handle for the memory pool.
         """
+        # Note: This is Linux only (int for file descriptor)
+        cdef int alloc_handle
```

Comment:
Do we need Windows support? FD's on Windows are pointer size. Should we introduce a conditional or make this data type a uintptr_t to ensure its large enough on all platforms?